packages/frontend2/src/app/(new)/(other)/scaling-next/summary/page.tsx

```diff
@@ -20,11 +21,44 @@ export const metadata = getDefaultMetadata({
 
 export default async function Page() {
   const tvl = await getTvl()
+  const hourly = (await getTvlChart({ range: '7d', type: 'layer2' })).map(
+    ([timestamp, native, canonical, external, ethPrice]) =>
+      [
+        timestamp,
+        (native + canonical + external) / 100,
+        native / 100,
+        canonical / 100,
+        external / 100,
+        (native + canonical + external) / ethPrice / 100,
+        native / ethPrice / 100,
+        canonical / ethPrice / 100,
+        external / ethPrice / 100,
+      ] as [
+        number,
+        number,
+        number,
+        number,
+        number,
+        number,
+        number,
+        number,
+        number,
+      ],
+  )
```

Comment:
Extract to separate function or/and add comments around this transformation - ik why it is there, somebody else might not